spec.bs

```diff
@@ -722,6 +692,7 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
 1. If |config|["{{AuctionAdConfig/resolveToConfig}}"] [=map/exists=]:
   1. Let |auctionConfig|'s [=auction config/resolve to config=] be
      |config|["{{AuctionAdConfig/resolveToConfig}}"].
+  1. TODO: What should happen if this rejects?

```

Comment:
typedef (USVString or FencedFrameConfig) UrnOrConfig;
Promise<UrnOrConfig?> runAdAuction(AuctionAdConfig config);

Yeah, impl-wise rejecting this also aborts the whole auction, which I guess could be done by setting this to failure and checking for that when accessing it, just like for others (unless we totally change the approach). 

... This one promise is a bit different since it's needed much, much, later than others, hence the "input promise" terminology in my helpers